openslides/users/static/js/users/site.js

```diff
@@ -620,6 +621,59 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                 // DS.flush();
             });
         };
+        $scope.openLoginForm = function () {
+            ngDialog.open({
+                template: 'static/templates/core/login-form.html',
+                controller: 'LoginFormCtrl',
+            });
+        };
+    }
+])
+
+.controller('LoginFormCtrl', [
+    '$scope',
+    '$http',
+    'operator',
+    'gettext',
+    'Config',
+    function ($scope, $http, operator, gettext, Config) {
+        $scope.alerts = [];
+
+        // TODO: add welcome message only on first time (or if admin password not changed)
+        $scope.alerts.push({
+            type: 'success',
+            msg: gettext("Installation was successfully.<br>" +
+                "Use <strong>admin</strong> and <strong>admin</strong> for first login." +
+                "<p>Important: Please change your password! ")
```

Comment:
Proposal: Try to remove the html-tags from the string, so it is not part of the translation.